src/utils/ssr/getAuthenticationInfo.js

```diff
@@ -2,28 +2,18 @@ import {
   CognitoIdentityClient,
   ListIdentityPoolsCommand,
 } from '@aws-sdk/client-cognito-identity';
-
 import {
   CognitoIdentityProviderClient,
   ListUserPoolsCommand,
   ListUserPoolClientsCommand,
   DescribeUserPoolClientCommand,
   DescribeUserPoolCommand,
 } from '@aws-sdk/client-cognito-identity-provider';
-
 import { getDefaultRoleAssumerWithWebIdentity } from '@aws-sdk/client-sts';
 import { fromTokenFile } from '@aws-sdk/credential-provider-web-identity';
 import configure from '../amplify-config';
-import Environment, { ssrGetCurrentEnvironment } from '../environment';
 
 const getAuthenticationInfo = async () => {
-  // if (
-  //   store.getState().networkResources.auth.userPoolId
-  //   || store.getState().networkResources.auth.identityPoolId
-  // ) {
-  //   return;
-  // }
-
```

Comment:
why do we not need this anymore?

Reply:
This was a relic from an old version when we saved this data to Redux, then I decided to not store it but just configure Amplify straight away, and this was left behind when I merged in my previous auth ticket.